float64[] q
---
bool status
